changelogs/fragments/3216-nmcli-idempotency-fixes.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - fixed idempotency issue when modifying bridge connections - module forces default value of bridge.priority to nmcli; when bridge.stp is disabled nmcli ignores is and keep default (https://github.com/ansible-collections/community.general/issues/3216, https://github.com/ansible-collections/community.general/issues/4683)
+  - fixed idempotency issue with 'may_fail4=false' passes when method4 is 'disabled'; in this case nmcli ignores change and keeps 'yes'.
```

Comment:
Every fragment must include a link to PR and/or issue. Also never use `yes` or `no` for booleans, always use `true` and `false`.

Reply:
In this case 'yes' - is a value provided by nmcli. I think I will paraphrase to make it clearer.
About PR, if I understand correct, to link fragment to PR it need create PR firstable then commit fragment change with actual link?

Reply:
> About PR, if I understand correct, to link fragment to PR it need create PR firstable then commit fragment change with actual link?

Yes, or you do some guessing (PR numbers are rather predictable). Doing it in two commits is easier though.

Reply:
I add links to fragments in PR that opened separately of this.